DOC_STRUCTURE.md

```diff
@@ -0,0 +1,88 @@
+# Documentation structure of the Babelfish Website
+
+## Documentation Index and folder structure
+
+The list of the documentation collection is configured in the `_config.yml` file, in the `just_the_docs` attribute. This is the original configuration:
+
+
+```yaml
+just_the_docs:
+  # Define the collections used in the theme
+  collections:
+    docs:
+    installation:
+      name: Installing Babelfish
+      nav_fold: true      
+    usage:
+      name: Using Babelfish
+      nav_fold: true
+    client:
+      name: Client programming
+      nav_fold: true
+    internals:
+      name: Babelfish internals
+      nav_fold: true
+    faq:
+      name: FAQ and getting help
+      nav_fold: true
+```
+
+Each one of the `collections` attributes points to the homonimous folder name, prefixed with an underscore:
+
+```
+internals -> _internals/
+client -> _client/
+```
+
+Each folder contains the documentation for each section, and they'll appear as subelements in the documentation sidebar. Each markdown file must contain the following header format:
```

Comment:
How about:

Each folder contains the documents that reside in that section; each document will appear as link in the navigation sidebar. Each markdown file must contain the following header format: